bigcommerce-labs/slate
# Updated dropdown menus to use radiobox hack

#### What?

Updated dropdowns to use radio button hack to show dropdowns on click instead of hover. 

#### How To Test

Click API and then Themes dropdowns. Check side nav layouts for inner docs pages. 

#### Screenshots (if appropriate)

![image](https://cloud.githubusercontent.com/assets/9373485/21694545/779574cc-d34b-11e6-9ef6-9a34de51aa84.png)
